From fa4904c9d9ad16e791485d0ef22566427869144c Mon Sep 17 00:00:00 2001
From: Xiang Xiao <xiaoxiang@xiaomi.com>
Date: Mon, 11 May 2020 00:03:51 +0800
Subject: [PATCH] system: nuttx: change clock_systimespec to
 clock_systime_timespec

follow up the change from NuttX side

Signed-off-by: Xiang Xiao <xiaoxiang@xiaomi.com>
---
 lib/system/nuttx/time.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/system/nuttx/time.c libmetal/lib/system/nuttx/time.c
index bafc957..986f1ba 100644
--- a/lib/system/nuttx/time.c
+++ libmetal/lib/system/nuttx/time.c
@@ -18,7 +18,7 @@ unsigned long long metal_get_timestamp(void)
 	struct timespec tp;
 	int r;
 
-	r = clock_systimespec(&tp);
+	r = clock_systime_timespec(&tp);
 	if (!r) {
 		t = (unsigned long long)tp.tv_sec * NSEC_PER_SEC;
 		t += tp.tv_nsec;
-- 
2.17.1

